client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+  const handleClick = function handleClick(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  };
```

Comment:
Own component could make sense (although it would make more sense if we start to build shared components folder). But I think this is a wrong place to wrap it as HOC.